CHANGES.md

```diff
@@ -5,14 +5,15 @@ Release Notes.
 8.11.0
 ------------------
 
-* Fix `cluster` and `namespace` value duplicated(`namespace` value) in properties report.
-* Add layer field to event when reporting.
-* Remove redundant `shade.package` property.
+
 
 #### Documentation
 
+* Add link about java agent injector.
+* Update configurations doc, remove `agent.instance_properties[key]=value`.
+* Update configurations doc, add `agent.cluster` and update `agent.namespace`.
 
-All issues and pull requests are [here](https://github.com/apache/skywalking/milestone/130?closed=1)
+All issues and pull requests are [here](https://github.com/apache/skywalking/milestone/120?closed=1)
```

Comment:
What is this change about? I think this is a mistake. And you should add your own change to changelog.

Reply:
This file is different every day. It always conflicts. I didn't change it. I just copied it from your main branch.

Reply:
The diff shows what you changed from main branch. You could see you put changes of others into Document section. You need to fix this.

Reply:
fine.But when we're done with the other issues, I'll revise this at the end. It's conflicted every day and I've changed it several times for nothing.

Reply:
Maybe so. If the downstream microservice is not registered, the error is irrelevant to Skywalking. The framework has its own logic processing. Maybe 1.x was written for that reason.

Reply:
Yes, it would be a conflict, it could be resolved when PR is ready to merge. But here, you change things, not conflict.